pkg/config/config.go

```diff
@@ -163,8 +152,17 @@ type Config struct {
 			Token           string
 			SupportsInvites bool `mapstructure:"supports_invites"`
 		}
-		LDAP *LDAP
-		OIDC OIDC
+		RemoteAuthenticator struct {
+			// Enabled if set true will enable remote authentication
+			Enabled bool `mapstructure:"enabled"`
+			// Endpoint URL of the remote authentication service (e.g. https://my-auth.example.com/auth)
+			Endpoint string `mapstructure:"endpoint"`
+			// DefaultUserGroup is the default group for the users authenticated by the remote service
+			DefaultUserGroup string `mapstructure:"default_user_group"`
+			// RequestTimeout timeout for remote authentication requests
+			RequestTimeout time.Duration `mapstructure:"request_timeout"`
+		} `mapstructure:"remote_authenticator"`
+		OIDC OIDC `mapstructure:"oidc"`
```

Comment:
I'm still confused with all of the OIDC references.

Reply:
In general OIDC has relevance in lakeFS, we support verifying **locally** tokens instead of going to an external service for that. So there is some configuration left over to verify the cookie. 

In the context of this PR, it's unrelated, I just added a missing `mapstructure:"oidc"` tag following up barack's comment in the config, could have been `blockstore.s3.credentials.access_key_id` or anything else.